### ISSUE TITLE
Error with master using x64

### ISSUE THREAD

**Issue by richardlittlej6** [openssl/openssl#6364]
I'm getting a not compatible error using Wins 7sp1 x64. Is there a x64 version available?

**Comment by levitte** [comment]
Did you build from source? Mind showing us how you configured? And oh, what version?

**Comment by richardlittlej6** [comment]
Well, I cloned the master repo, then I tried the commands available on openSSL domain for Windows. After loading the config file using cmd an error came back "not compatible with the version of Windows that I am running". Also asked if there were a x64 version of the software available

**Comment by levitte** [comment]
Sorry, didn't read the title right. So Master, thanks.

The source should build on x64. In fact, it does for us... but I still need to see how you configured, "loading the config file using cmd" means nothing to me, I need to see the exact command line you used to be able to tell you what's going on.

**Comment by richardlittlej6** [comment]
![err](https://user-images.githubusercontent.com/21162712/40596362-d477002a-6229-11e8-9346-c9a1c1c17c58.PNG)

**Comment by levitte** [comment]
Those look like git commands, except you forgot to start the command with `git`, as in `git config core.autocrlf false`.

However, to configure *OpenSSL*, you should really read read `INSTALL`. Here's a quick start:

```
perl Configure VC-WIN64A
```

Now, this is actually not quite the right forum for these questions, as issues here are supposed to be about actual bugs and feature requests, not generic Q&A, so I suggest you turn to our mailing list openssl-users. See https://www.openssl.org/community/mailinglists.html for subscription info.

I'm closing this now.

**Comment by richardlittlej6** [comment]
Well, this program definitely works best with Perl for x64bit Windows, however I am getting numerous errors for the file (path: openssl\Configurations) windows-makefile template  that need debugged. Also installs fine with cynwin.

**Comment by levitte** [comment]
Well, if you're gonna use things completely wrong, expect wrong results.

Please  Read `INSTALL`!  Or perhaps just follow the example configuration I mentioned: `perl Configure VC-WIN64A`

**Comment by richardlittlej6** [comment]
Yeah, thanks I did that add this is the error I got. After trying a couple different method then working a some of bugs out, I got a commit that prompted my to try a different method again. I used the VC-WIN64-AMP prompt, that one worked then I scrapped the change debug file for the original of course the new prompt gave me a different template that word with all the new SSL or should I say TLS capabilities. Thanks for the pointers though
